frontend/src/services/userAuthService.ts

```diff
@@ -14,44 +15,56 @@ export interface UserCredentials {
     password: string;
 }
 
-export const submitUserData = async (userData: UserData) => {
-    try {
-        const response = await axios.post('http://34.172.117.230:8080/api/v1/account', userData);
+export const userAuthService = {
+    register: async (userData: UserData) => {
+        try {
+            const response = await axios.post('/v1/account', userData);
+            const { accessToken } = response.data;
+
+            if (!accessToken) {
+                throw new Error('Nieprawidłowe dane rejestracji.')
+            }
 
-        if (!response.data) {
-            throw new Error('Nieprawidłowe dane rejestracji.')
+            tokenService.setToken(accessToken);
```

Comment:
To samo co wcześniej, czyli przy rejestracji nie zwracamy tokenu (ale tak samo jak wcześniej sprawdź to jeszcze dla pewności), więc nie będziemy ustawiać tutaj tokenu